packages/definitions/src/fieldType.ts

```diff
@@ -79,6 +80,21 @@ export const FIELD_TYPE = Object.freeze({
   GEOMETRY_MULTILINESTRING: 'GEOMETRY_MULTILINESTRING',
 })
 
+/**
+ * Component to use for a field
+ */
+export const FIELD_COMPONENT = Object.freeze({
+  INPUT_TEXT: 'INPUT_TEXT',
+  INPUT_PASSWORD: 'INPUT_PASSWORD',
+  INPUT_NUMBER: 'INPUT_NUMBER',
+  INPUT_FLOAT: 'INPUT_FLOAT',
+  INPUT_CURRENCY: 'INPUT_CURRENCY',
+  INPUT_DATE: 'INPUT_DATE',
+  INPUT_DATETIME: 'INPUT_DATETIME',
+  INPUT_EMAIL: 'INPUT_EMAIL',
+  INPUT_UUID: 'INPUT_UUID',
```

Comment:
I would like to suggest that we sort this kind of list alphabetically.

Reply:
done